### DISCUSSION TITLE
Join meetings

### DISCUSSION THREAD

**Discussion by llhardy1A** [Green-Software-Foundation/if#76] [General]
Hi everyone,
I'm member of the GSF for at least few months now :), following the standards and the patterns but, due to an internal project in my company, I would like to follow the progress on CarbonQL and so to attend to your meetings.
Can someone forward me the invitations please?
Do not hesitate to ping me directly if you have comment or questions, thank you

**Comment by jawache** [comment]
Hia @llhardy1A the best way is to fill out this form: https://greensoftware.foundation/subscribe/ and it should add you to everything automatically (you must have called the register form previously though https://greensoftware.foundation/onboarding/)

**Reply by llhardy1A** [reply]
Hello,
Ok great, thank you very much. I'm going to have a look.
Have a great day,
Leila
